docs/10-variables/01-overview/index.md

```diff
@@ -26,8 +26,7 @@ Variables can be used throughout the component or page:
 
 This makes variables ideal for managing temporary states like:
 
-- UI state (expanded/collapsed, selected tabs)
-- Performance-sensitive interactions
```

Comment:
I kind of liked this point though

Reply:
But I don't really understand what it means. It just sounds like jargon 😅

Reply:
Perhaps it could say: `- Performance-sensitive or frequent state updates`
I'm also okay with removing it

Reply:
But what does performance-sensitive mean in this context?